jenkins-builds/production_builds.sh

```diff
@@ -50,72 +108,75 @@ eb --show-config
-  eb $build -r --modules-header=$APPS/UES/login/daint-${ARCH}.h --modules-footer=${EASYBUILD_TMPDIR}/${name}.footer
-# change permissions for selected builds (note that $USER needs to be member of the group to use the command chgrp)
-  echo -e "\n Changing group ownership and permissions for ${name} folders:\n - ${EASYBUILD_PREFIX}/software/${name}"
-  chgrp ${name,,} -R ${EASYBUILD_INSTALLPATH}/software/${name}
-  chmod -R o-rwx ${EASYBUILD_INSTALLPATH}/software/${name}/*
-# standard build without need to add a module footer or adjusting ownership and permissions
- else
-  echo -e "eb $build -r --modules-header=$APPS/UES/login/daint-${ARCH}.h"
-  eb $build -r --modules-header=$APPS/UES/login/daint-${ARCH}.h
- fi
-
-# create default
- echo -e "\n Creating file ${EASYBUILD_INSTALLPATH}/modules/all/${name}/.version to set ${version} as default for ${name}"
- cat > ${EASYBUILD_INSTALLPATH}/modules/all/${name}/.version<<EOF
-#%Module
-set ModulesVersion "${version}"
-EOF
+   fi
+    echo -e "eb $build -r ${eb_args} ${tmp_footer}\n"
+    eb $build -r ${eb_args} ${tmp_footer}
+
+    # change permissions for selected builds (note that $USER needs to be member of the group to use the command chgrp)
+    echo -e "\n Changing group ownership and permissions for ${name} folders:\n - ${EASYBUILD_PREFIX}/software/${name}"
+    chgrp ${name,,} -R ${EASYBUILD_INSTALLPATH}/software/${name}
+    chmod -R o-rwx ${EASYBUILD_INSTALLPATH}/software/${name}/*
+
+  #
+  # The other builds
+  #
+  else
+    echo -e "eb $build -r ${eb_args}"
+    eb $build -r ${eb_args}
+  fi
+
 done
 
-# update xalt table of modulefiles
-echo "loading PrgEnv-cray"
-module load PrgEnv-cray/6.0.3
-
-echo "module use craypat apps"
-module use /apps/daint/UES/6.0.UP02/craypat/easybuild/modules/all
-
-# Removing Easybuild module before the reverseMapD operation
-# because spider the mapping the programs to Easybuild
-module unload Easybuild
-
-echo "running reverseMapD"
-userid=`id -u`
-if [ "X$userid" == "X23395" ] && [ "X$hostName" == "Xdaint" ]; then
-	module load Lmod
-	export PATH=$EBROOTLMOD/lmod/7.1/libexec:$PATH  # !!! for spider !!!
-	export XALTJENKINS=/apps/daint/UES/xalt/JENSCSCS
-	export XALTPROD=/apps/daint/UES/xalt/git
-	cd $XALTJENKINS/
-	rm -rf $XALTJENKINS/reverseMapD
-	./cray_build_rmapT.sh .
-	cp ./reverseMapD/*    $XALTPROD/etc/reverseMapD/
-	cd -
+
+if [ "Xdaint" == "X$hostName" ]; then
+    # update xalt table of modulefiles
+    echo "loading PrgEnv-cray"
+    module load PrgEnv-cray/6.0.3
+    echo "module use craypat apps"
+    module use /apps/daint/UES/6.0.UP02/craypat/easybuild/modules/all
+    # Removing Easybuild module before the reverseMapD operation
+    # because spider the mapping the programs to Easybuild
+    module unload Easybuild
+    echo "running reverseMapD"
+    userid=`id -u`
+    if [ "X$userid" == "X23395" ] && [ "X$hostName" == "Xdaint" ]; then
```

Comment:
at this point you have already tested `"Xdaint" == "X$hostName"`, why checking again?